sdk/communication/azure-communication-chat/swagger/README.md

```diff
@@ -37,7 +37,7 @@ output-folder: ..\
 license-header: MICROSOFT_MIT_SMALL
```

Comment:
Do not use a local swagger unless you have to. Publish the swagger before regenerating the clients. ReadMe.md should only refer to published version of the swagger

Reply:
We don't have a published swagger to reference at this point

Reply:
we can ask @amrelroumy-msft to publish one for us. All sdks need it now.